ui/scss/_gui.scss

```diff
@@ -188,3 +188,31 @@ section.section-spaced {
 {
   text-align: center;
 }
+
+.error-text {
+  color: var(--color-error);
+}
+
+.display-inline-block {
```

Comment:
I think these additions are likely not necessary. Just put on the components as relevant.